src/modules/app_commands/sendHttpRequest.js

```diff
@@ -0,0 +1,47 @@
+/**
+ * Payload for sending http(s) request via app command.
+ *
+ * @param {string} url Http url.
+ * @param {string} serial Name the request.
+ * @param {*} [body] Parameter to the request.
+ * @param {string} [method='POST'] Http method.
+ * @param {string} [contentType='application/x-www-form-urlencoded'] ContentType header.
+ * @param {number} [timeout=1000] Request timeout.
+ *
+ * @return {AppCommand}
+ * @since lib 13.0
+ */
+export function sendHttpRequest({
+  url,
+  serial,
+  body,
+  method = 'POST',
+  contentType = 'application/x-www-form-urlencoded',
+  timeout = 10000,
+  followRedirects = false,
+  synchronizeCookies = true,
```

Comment:
Could you please add jsdoc for followRedirects and synchronizeCookies

Reply:
Done